addons/maptools/functions/fnc_handleMouseButton.sqf

```diff
@@ -72,44 +76,100 @@ if (_dir != 1) then {
         GVAR(mapTool_isRotating) = false;
         _handled = true;
     };
+    if (GVAR(plottingBoard_isDragging) || GVAR(plottingBoard_isRotating > -1)) then {
+        GVAR(plottingBoard_isDragging) = false;
+        GVAR(plottingBoard_isRotating) = -1;
+        _handled = true;
+    };
 } else {
     // If clicking
-    if !(call FUNC(canUseMapTools)) exitWith {};
-
-    // Transform mouse screen position to coordinates
-    private _pos = _control ctrlMapScreenToWorld [_screenPosX, _screenPosY];
-    _pos set [count _pos, 0];
-
-    GVAR(mapTool_isDragging) = false;
-    GVAR(mapTool_isRotating) = false;
+    if (call FUNC(canUseMapTools)) then {
 
-    // If no map tool marker then exit
-    if (GVAR(mapTool_Shown) == 0) exitWith {};
+        // Transform mouse screen position to coordinates
+        private _pos = _control ctrlMapScreenToWorld [_screenPosX, _screenPosY];
+        _pos set [count _pos, 0];
```

Comment:
As of 'Arma 3' 2.12 you can use negative values instead of `count ...`, like `-1` for the last element, or `-2` for 2nd last, etc.:
```suggestion
        _pos set [-1, 0];
```

Reply:
`count _pos` would be 1 past the end of the vector not the last
```suggestion
        _pos pushBack 0;
```
is probably what we want